client/src/components/App.jsx

```diff
@@ -100,13 +108,13 @@ const App = () => {
   }
 
   return (
-    <Router>
+    <>
       <h1>App.jsx</h1>
       <Overview objID={id} yourOutfit={yourOutfit} addToOutfit={addToOutfit}/>
       <Questions objID={id}/>
       <Ratings objID={id}/>
```

Comment:
<Ratings objID={id} generateStars={generateStars}/>
  
  Need to pass generateStars down to the Ratings component

Reply:
<Ratings objID={id} generateStars={generateStars}/>

Reply:
Made this change!